server/server.go

```diff
@@ -105,6 +105,10 @@ func NewServer(opts ...server_options.ServerOption) *Server {
 		panic(err)
 	}
 
+	e.GET("/healthz", func(c echo.Context) error {
```

Comment:
could you put this route in a server/routes/health.go and follow the same pattern as the other routes are set

Reply:
Done :)

Reply:
makes sense, still want to make the `/route` folder help with the high level picture of the available routes